Sources/App/Core/CFRayRouteLoggingMiddleware.swift

```diff
@@ -0,0 +1,17 @@
+import Vapor
+
+// Replica of the Vapor RouteLoggingMiddleware that's tweaked to explicitly expose the cf-ray header in the logger metadata for the request.
+public final class CFRayRouteLoggingMiddleware: Middleware {
+    public let logLevel: Logger.Level
+    
+    public init(logLevel: Logger.Level = .info) {
+        self.logLevel = logLevel
+    }
+    
+    public func respond(to request: Request, chainingTo next: Responder) -> EventLoopFuture<Response> {
+        let cfray = request.headers.first(name: "cf-ray") ?? "--nil--"
```

Comment:
Is there a reason you want to track nils in the logs? Or should we rather just skip them?

```suggestion
        guard let cfray = request.headers.first(name: "cf-ray") else {
            return next.respond(to: request)
        }
```

Reply:
Also, I think it'll be useful to be able to switch this on and off with a simple env variable toggle that only requires a variable change + redeployment without a full new release build.

I'll open a PR on top of yours with suggested changes!